telicent_lib/adapter.py

```diff
@@ -140,6 +139,8 @@ def __init__(self, target: DataSink, adapter_function: RecordAdapter,
         :param name: The name of the Adapter
         :type name: str
         :param source_name: The name of the data source, used in the startup banner
+        :param pbac_obj: An instance of Security policy model populated with data
+        :type object
```

Comment:
```suggestion
        :type pbac_job: object
```

Just to match the other styling